create_compare_branch.sh

```diff
@@ -0,0 +1,15 @@
+git fetch
+git checkout shrestha/Add_CompilerFlag_VariableChange
+
+mkdir temp
+cp -a ./src/enable_variable_ops/* ./temp
+
+git checkout -
+
+cp -a temp/* ./src
+
+git add ./src/*
+git commit -am "Diff changes"
+
+git push
+
```

Comment:
running this script will download PR#479 copy the files of `src/enable_variable_ops` into `src` and push to this branch `sarkars/compare_tensor_hiding`, so that the diffs are more visible